### ISSUE TITLE
JDL with multiline comment not close: generating dirty code without any error

### ISSUE THREAD

**Issue by funder7** [jhipster/generator-jhipster#12837]
##### **Overview of the issue**

I did a mistake by not closing a multiline comment in a app JDL entity definition, the app was generated successfuly but some fields after the missed comment closure where missing. 

##### **Reproduce the error**

```jdl
entity Payment {
  /** My payment date */
  paymentDate Instant
  /** True when amount has been paid (captured payments)   <------- not closed
  paid Boolean
  currency Currency required
  /** Amount to be collected  */
  amount Integer required min(42)
  description String maxlength(127)
  /** Response from Stripe service */
  status String required
  /** Optional transaction data to be stored */
  receipt TextBlob
}
```

##### **Suggest a Fix**

Maybe the multiline comment can be underlined with warning style, when it contains a type (like "Boolean") 

##### **JHipster Version(s)**

6.10.4

##### **JHipster configuration**

```jdl
application {
  config {
    applicationType monolith
    authenticationType oauth2
    baseName xxxxx
    buildTool maven
    blueprints [ generator-jhipster-primeng-blueprint]
    cacheProvider ehcache
    clientFramework angularX
    clientPackageManager npm
    clientTheme materia
    clientThemeVariant primary
    databaseType sql
    devDatabaseType postgresql
    dtoSuffix DTO
    enableHibernateCache true
    enableSwaggerCodegen true
    enableTranslation true
    languages [en, it, fr, es, de]
    messageBroker false
    nativeLanguage it
    prodDatabaseType postgresql
    searchEngine false
    serverPort 8080
    serviceDiscoveryType false
    skipClient false
    skipServer false
    skipUserManagement true
    testFrameworks [protractor]
    useSass true
    websocket spring-websocket
  }

}
```

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by mshima** [comment]
I don't see this as a bug.
As you can see it was closed by `/** Amount to be collected  */`.
```
  /** True when amount has been paid (captured payments)   <------- not closed
  paid Boolean
  currency Currency required
  /** Amount to be collected  */
```
The only thing that can be done is a warning with `\**` inside another multi-line comment ` \** \** *\`.
IMO is not worth.

**Comment by pascalgrimaud** [comment]
no need warning as it can be commented ntentionaly

For me, it's like in your own code, if you forgot to close comment, your code won't be included.

Closing this as it's not a bug and nothing we can do in our side

**Comment by funder7** [comment]
Ok, no problem, you're right. May I ask a question before opening a new issue? 
I've generated a project different times, starting from the same JDL with minor changes here and there. I noticed that many Swagger annotations were not present anymore when doing another import. 
I don't know how to reproduce this...any idea on why it happens?

**Comment by pascalgrimaud** [comment]
> I noticed that many Swagger annotations were not present anymore when doing another import.

I don't understand what you mean by swagger annotation

**Comment by MathieuAA** [comment]
Are the projects generated with the same JHipster version?

**Comment by funder7** [comment]
@pascalgrimaud I meant these two:

`@ApiModel(description = "......")` on classes
`@ApiModelProperty(value = ".....")` on fields

They're added when you comment the jdl: 

```
/**
 * Bag depot - where bags are stored
 */
entity BagDepot {
  lat BigDecimal
  lng BigDecimal
  /**
   * Depot Name
   */
  name String required maxlength(120)
  /**
   * If this depot is active or not
   */
  active Boolean
}
```

@MathieuAA yes I started with an earlier version, then did an upgrade. But to say it all, I created the project from scratch (using the same jdl - let's call it' B) and then merged the changes in my project. Some annotations are not present in the B project.
